### ISSUE TITLE
Mouse click resize the windows.

### ISSUE THREAD

**Issue by 4goodapp** [cyrus-and/zoom#23]
I am having this weird behavior where making mouse click resize the windows (till you release the button). like in this gif:

https://i.ibb.co/Pg180XC/Peek-2019-03-12-16-51.gif

**Comment by cyrus-and** [comment]
What version of Emacs are you using?

**Comment by 4goodapp** [comment]
GNU Emacs 26.1 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.22.30) of 2018-07-05

**Comment by cyrus-and** [comment]
Thanks, and does this happen always or sometimes only? Also, can you reproduce it with `emacs -Q`?

**Comment by 4goodapp** [comment]
It only happens sometimes. Like 70%
in `emacs -Q` , far less and it happens just once . seems like it has something to do with the mark but am not really sure. 
And btw, I don't have that issue with golden ratio.

**Comment by cyrus-and** [comment]
That's weird, I have:

```
GNU Emacs 26.0.90 (build 1, x86_64-pc-linux-gnu, GTK+ Version 2.24.31) of 2017-11-27
```

If I run Emacs like:

```console
command emacs -Q \
        --eval '(load "/path/to/zoom/zoom.el")' \
        --eval '(zoom-mode)' \
        --eval '(split-window-right)' \
        --eval '(switch-to-buffer "*Messages*")'
```

And I start clicking in both windows I cannot reproduce the issue.

Besides, I always use Zoom in my everyday tasks and it never happens. It used to happen though, if I recall correctly, with some prior Emacs/Zoom versions. Are you sure you're using the the latest Zoom version? Just to be sure, if you're using MELPA, clone this repo and use that.

FYI this is the relevant Elisp part:

https://github.com/cyrus-and/zoom/blob/2cdb15c9e753222c8600120edecfb46c2fbfae5d/zoom.el#L170-L180

**Comment by 4goodapp** [comment]
Thanks for the detailed reply.
I can't reproduce it either with `emacs -Q`. 
That's weird.

I'll ask a friend to test it and see if the issue repeats. meanwhile I think you can close the issue.

**Comment by cyrus-and** [comment]
If you manage to find the mode that conflicts with Zoom let me know and I'll look into it.

Thanks for your patience. Closing for now...

**Comment by cyrus-and** [comment]
FYI possibly fixed in v0.2.2, whatever it was...

**Comment by 4goodapp** [comment]
Seems like it's fixed in the latest update. 
Thanks!
